internal/cli/atlas/performanceadvisor/slowquerylogs/list.go

```diff
@@ -39,7 +39,7 @@ type ListOpts struct {
 	store      store.PerformanceAdvisorSlowQueriesLister
 	since      int64
 	duration   int64
-	namespaces string
+	namespaces []string
```

Comment:
[q] Is this a breaking change for users, or was no data being returned previously, so they never saw a string value here?

Reply:
this is internal and has no impact to users except they used to do `--namespaces` once, they can do it multiple times now